client/src/components/JurisdictionAdmin/generateSheets.ts

```diff
@@ -795,3 +797,128 @@ export const downloadTallyEntryLoginLinkPrintout = async (
   )
   return doc.output() // returned for test snapshots
 }
+
+const AUDIT_BOARD_SIGN_MAX_CONTEST_NAME_LENGTH = 40
+const AUDIT_BOARD_SIGN_MAX_SURNAME_LENGTH = 20
+const AUDIT_BOARD_SIGN_TITLE_LENGTH_XL_FONT_CUTOFF = 12
+const AUDIT_BOARD_SIGN_TEXT_COLOR = Colors.GRAY3
+const AUDIT_BOARD_SIGN_TITLE_COLORS = [
+  Colors.GREEN1,
+  Colors.VIOLET1,
+  Colors.ORANGE1,
+  Colors.VERMILION1,
+  Colors.BLUE1,
+  Colors.COBALT1,
+  Colors.GOLD1,
+  Colors.SEPIA1,
+  Colors.DARK_GRAY1,
+  Colors.INDIGO1,
+]
+const DEFAULT_AUDIT_BOARD_SIGNS = [
+  { title: 'No Vote', subtitle: '(undervote, overvote, blank, etc.)' },
+  { title: 'For Review', subtitle: '(undetermined, duplicated, etc.)' },
+]
+
+function parseSurname(fullName: string, maxLength: number) {
+  const suffixes = ['sr.', 'sr', 'jr.', 'jr', 'ii', 'iii', 'iv', 'v']
+  const nameParts = fullName.trim().split(' ')
+  const lastNameIndex = suffixes.includes(
+    nameParts[nameParts.length - 1].toLowerCase()
+  )
+    ? nameParts.length - 2
+    : nameParts.length - 1
+  let surname = nameParts.slice(lastNameIndex).join(' ')
+  if (surname.length > maxLength) {
+    surname = `${surname.slice(0, maxLength)}...`
+  }
+  return surname
+}
+
+function formCandidateSigns(
+  contests: IMinimalContest[]
+): { title: string; subtitle: string }[] {
+  const candidateSigns: { title: string; subtitle: string }[] = []
+  contests.forEach(contest => {
+    const contestName =
+      contest.name.length > AUDIT_BOARD_SIGN_MAX_CONTEST_NAME_LENGTH
+        ? `${contest.name.slice(
+            0,
+            AUDIT_BOARD_SIGN_MAX_CONTEST_NAME_LENGTH
+          )}...`
+        : contest.name
+    contest.choices.forEach(choice => {
+      const candidate = {
+        title: parseSurname(choice.name, AUDIT_BOARD_SIGN_MAX_SURNAME_LENGTH),
+        subtitle: `(Contest: ${contestName})`,
+      }
+      candidateSigns.push(candidate)
+    })
+  })
+  return candidateSigns
+}
+
+export const downloadAuditBoardSigns = async (
+  auditName: string,
+  contests: IMinimalContest[],
+  jurisdictionName: string
+): Promise<string> => {
+  const doc = new jsPDF({ format: 'letter', unit: 'pt' })
+
+  const candidateSigns = formCandidateSigns(contests)
+  const allSigns = DEFAULT_AUDIT_BOARD_SIGNS.concat(candidateSigns)
+  for (let i = 0; i < allSigns.length; i += 1) {
+    if (i !== 0) {
+      doc.addPage()
+    }
+    const sign = allSigns[i]
+    const titleColor =
+      AUDIT_BOARD_SIGN_TITLE_COLORS[i % AUDIT_BOARD_SIGN_TITLE_COLORS.length]
+
+    // Mid-page fold
+    doc.setFont('Helvetica', 'normal').setFontSize(defaultFontSize)
+    doc.setTextColor(AUDIT_BOARD_SIGN_TEXT_COLOR)
+    const foldCaption = 'Fold here and place at edge or table or tray'
+    let x = pageMargin / 2
+    let y = pageHeight / 2
+    doc.text(foldCaption, x, y)
```

Comment:
Should we add a dotted line? It's a classic fold signifier

Reply:
Good call!